modules/bluefish/CMakeLists.txt

```diff
@@ -3,18 +3,18 @@ project (bluefish)
 
 set(SOURCES
 		consumer/bluefish_consumer.cpp
-
 		util/blue_velvet.cpp
-
 		bluefish.cpp
 		StdAfx.cpp
 )
 set(HEADERS
+		interop/BlueDriver_p.h
+		interop/BlueHancUtils.h
+		interop/BlueVelvetC.h
+		interop/BlueVelvetCUtils.h
```

Comment:
I was simply copying what was done with the blackmagic module - Which may or may not have been a good idea.  I am happy to move them to dependencies64/bluefish/include, As i personally feel that is probably the correct location for the files.

Personally i tend to agree that